apps/admin-gui/src/app/users/pages/user-detail-page/user-attributes/user-attributes.component.ts

```diff
@@ -48,6 +55,7 @@ export class UserAttributesComponent implements OnInit {
   selection = new SelectionModel<Attribute>(true, []);
   attributes: Attribute[] = [];
   userId: number;
+  facilities: Facility[] = [];
```

Comment:
dunno why you added this, it doesnt seems like you are using it somewhere
if thats the case please delete also unused things from constructor and import